core/dbt/contracts/util.py

```diff
@@ -207,13 +208,45 @@ def get_manifest_schema_version(dct: dict) -> int:
     return int(schema_version.split(".")[-2][-1])
 
 
+# we renamed these properties in v1.3
+# this method allows us to be nice to the early adopters
+def rename_metric_attr(data: dict, raise_deprecation_warning: bool = False) -> dict:
+    if raise_deprecation_warning and "sql" in data.keys() or "type" in data.keys():
+        deprecations.warn("metric-attr-renamed")
+    duplicated_attribute_msg = """
+Cannot define both '{}' (old name) and '{}' (new name).
+These are the same attribute. Remove the first and leave the second.
```

Comment:
@callum-mcdata Very open to your wordsmithing on the exception here...

Reply:
Love it. I'm just going to change `parameter` &rarr; `property` because that seems most consistent with what we call these things in our docs today